builtin/providers/openstack/resource_openstack_lbaas_loadbalancer_v2.go

```diff
@@ -0,0 +1,258 @@
+			},
+		},
+	}
+}
+
+func resourceLoadBalancerV2Create(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	networkingClient, err := config.networkingV2Client(d.Get("region").(string))
+	if err != nil {
+		return fmt.Errorf("Error creating OpenStack networking client: %s", err)
+	}
+
+	adminStateUp := d.Get("admin_state_up").(bool)
+	createOpts := loadbalancers.CreateOpts{
+		Name:         d.Get("name").(string),
+		Description:  d.Get("description").(string),
+		VipSubnetID:  d.Get("vip_subnet_id").(string),
+		TenantID:     d.Get("tenant_id").(string),
+		VipAddress:   d.Get("vip_address").(string),
+		AdminStateUp: &adminStateUp,
+		Flavor:       d.Get("flavor").(string),
+		Provider:     d.Get("provider").(string),
+	}
+
+	log.Printf("[DEBUG] Create Options: %#v", createOpts)
+	lb, err := loadbalancers.Create(networkingClient, createOpts).Extract()
+	if err != nil {
+		return fmt.Errorf("Error creating OpenStack LoadBalancer: %s", err)
+	}
+	log.Printf("[INFO] LoadBalancer ID: %s", lb.ID)
+
+	log.Printf("[DEBUG] Waiting for Openstack LoadBalancer (%s) to become available.", lb.ID)
+
+	stateConf := &resource.StateChangeConf{
+		Pending:    []string{"PENDING_CREATE"},
+		Target:     []string{"ACTIVE"},
+		Refresh:    waitForLoadBalancerActive(networkingClient, lb.ID),
+		Timeout:    2 * time.Minute,
+		Delay:      5 * time.Second,
+		MinTimeout: 3 * time.Second,
+	}
+
+	_, err = stateConf.WaitForState()
+	if err != nil {
+		return err
+	}
+
+	d.SetId(lb.ID)
+
+	return resourceLoadBalancerV2Read(d, meta)
+}
+
+func resourceLoadBalancerV2Read(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	networkingClient, err := config.networkingV2Client(d.Get("region").(string))
+	if err != nil {
+		return fmt.Errorf("Error creating OpenStack networking client: %s", err)
+	}
+
+	lb, err := loadbalancers.Get(networkingClient, d.Id()).Extract()
+	if err != nil {
+		return CheckDeleted(d, err, "LoadBalancerV2")
+	}
+
+	log.Printf("[DEBUG] Retreived OpenStack LoadBalancerV2 %s: %+v", d.Id(), lb)
+
+	d.Set("name", lb.Name)
+	d.Set("description", lb.Description)
+	d.Set("vip_subnet_id", lb.VipSubnetID)
+	d.Set("tenant_id", lb.TenantID)
+	d.Set("vip_address", lb.VipAddress)
+	d.Set("admin_state_up", lb.AdminStateUp)
+	d.Set("flavor", lb.Flavor)
+	d.Set("provider", lb.Provider)
+
+	return nil
+}
+
+func resourceLoadBalancerV2Update(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	networkingClient, err := config.networkingV2Client(d.Get("region").(string))
+	if err != nil {
+		return fmt.Errorf("Error creating OpenStack networking client: %s", err)
+	}
+
+	var updateOpts loadbalancers.UpdateOpts
+	if d.HasChange("name") {
+		updateOpts.Name = d.Get("name").(string)
+	}
+	if d.HasChange("description") {
+		updateOpts.Description = d.Get("description").(string)
+	}
+	if d.HasChange("admin_state_up") {
+		asu := d.Get("admin_state_up").(bool)
+		updateOpts.AdminStateUp = &asu
+	}
```

Comment:
`admin_state_up` is configured with `ForceNew: true`, so this block here would never be evaluated. I'd say either `ForceNew` or this block needs removed.